src/sentry/static/sentry/app/components/events/interfaces/spans/utils.tsx

```diff
@@ -307,6 +307,8 @@ export const setBodyUserSelect = (nextValues: UserSelectValues): UserSelectValue
     // MozUserSelect is not typed in TS
     // @ts-ignore
     MozUserSelect: document.body.style.MozUserSelect,
+    // msUserSelect is not typed in TS
+    // @ts-ignore
```

Comment:
msUserSelect is now also missing in ts 4

Reply:
Any idea why? Should we be dropping support too?

Reply:
https://caniuse.com/user-select-none

Seems like we could

Reply:
Let's keep this. 

I'm aware of this being a missing type in TypeScript. This was causing an issue now because of updated type definitions for user select on v4.